cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceAction.scss

```diff
@@ -123,12 +130,17 @@ $link-color: #0275d8;
                 overflow: scroll;
 
                 .key-value-pair-preference {
+                  margin: 0;
                   justify-content: initial;
 
                   input {
                     border: 1px solid #cccccc;
                     width: 36%;
                     margin: 5px 50px 15px 0px;
+
+                    &:focus {
+                      border-color: $focused-input-color;
```

Comment:
This also needs a margin top I think. Because of lack of margin top the first row looks like this,

<img width="882" alt="screen shot 2017-01-30 at 11 43 02 am" src="https://cloud.githubusercontent.com/assets/1452845/22438620/8e117e54-e6e1-11e6-89d3-bf5d9767ecee.png">